outplacement/__manifest__.py

```diff
@@ -32,7 +32,7 @@
     'author': 'Vertel AB',
     'license': 'AGPL-3',
     'website': 'http://www.vertel.se',
-    'depends': ['base', 'hr', 'mail'],
+    'depends': ['base', 'hr', 'mail', 'partner_firstname'],
```

Comment:
I'm not sure that we need this depends here. You have to add first_name and last_name manually with relations to res partner

Reply:
I wasn't sure where to put it, but it's necessary for the completion report API and probably others as well so I thought it'd make sense to be here.

Reply:
If completion report API needs this, can this be put in the manifest of completion report?

Reply:
I guess, just seems a bit strange to have it as a dependency in every api module that happens to need it.